.github/workflows/ci.yaml

```diff
@@ -0,0 +1,108 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - use-github-actions
+
+env:
+  registry: ghcr.io
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: angular
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - name: Check out the repository
+      id: checkout
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+    - name: Set up node
+      id: setup-node
+      uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
+      with:
+        node-version: current
+        cache: npm
+        cache-dependency-path: angular/package-lock.json
+
+    - name: Run clean install
+      id: npm-ci
+      run: npm ci
+
+    - name: Run tests
+      id: npm-test
+      run: npm test -- --browsers ChromeHeadless --no-watch --no-progress
```

Comment:
Why not to use `npm run test` command? https://github.com/kuwairo/angular-react-starter/blob/master/angular/package.json#L10

Reply:
In this case they seem to be equivalent, but for the sake of consistency I have now changed it to “run”.